tgui/packages/tgui/interfaces/GasMixer/MixInfo.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * @file
+ * @copyright 2024
+ * @author Garash2k (https://github.com/garash2k)
+ * @license MIT
+ */
+
+import type { airInfo } from './types';
+
+interface MixInfoProps {
+  mix: airInfo;
+}
+export const MixInfo = (props: MixInfoProps) => {
+  const { mix } = props;
+  return (
+    <>
+      {mix.gasses
+        .filter((gas) => gas.Ratio > 0)
+        .sort((gas) => gas.Ratio)
+        .reverse()
+        .map((gas) => (
+          <div key={gas.Name} style={{ color: gas.Color }}>
+            {gas.Name}: {gas.Ratio}%
+          </div>
+        ))}
+      <br />
+      {mix.kpa && `Pressure: ${mix.kpa} kPa`}
+      <br />
+      {mix.temp && `Temperature: ${mix.temp} °C`}
```

Comment:
There's got to be a better way of displaying this. 🤔 `LabeledList`?

Reply:
For sure, like I said in the draft points, MixInfo's looks are far from final.